### ISSUE TITLE
Fix `validate_db_data_with_read_schemas.py`

### ISSUE THREAD

**Issue by tcompa** [fractal-analytics-platform/fractal-server#1227]
We have this bunch of logs:
```python traceback
Project 17 validated
Project 18 validated
Traceback (most recent call last):
  File "/home/runner/work/fractal-server/fractal-server/scripts/validate_db_data_with_read_schemas.py", line 80, in <module>
    ApplyWorkflowRead(**job.model_dump())
  File "pydantic/main.py", line 341, in pydantic.main.BaseModel.__init__
pydantic.error_wrappers.ValidationError: 3 validation errors for ApplyWorkflowRead
workflow_dump -> timestamp_created
  field required (type=value_error.missing)
input_dataset_dump -> timestamp_created
  field required (type=value_error.missing)
output_dataset_dump -> timestamp_created
  field required (type=value_error.missing)
Project 20 validated
Project 23 validated
Project 24 validated
```

Why are we still validating projects after one validation fails?

**Comment by ychiucco** [comment]
https://github.com/fractal-analytics-platform/fractal-server/actions/runs/7888861060/job/21527339027

It seems that the script fails correctly where it should, and that the problem is only related to how the GitHub runner prints the output.

The last lines printed are these
```sh
Dataset 86 validated
Dataset 87 validated
Dataset 88 validated
Dataset 89 validated
Dataset 90 validated
Dataset 96 validated
Dataset 102 validated
Dataset 119 validated
Dataset 120 validated
Dataset 121 validated
Dataset 122 validated
Dataset 126 validated
Dataset 127 validated
Dataset 130 validated
Dataset 131 validated
Error: Process completed with exit code 1.
```

 so the script executes this line https://github.com/fractal-analytics-platform/fractal-server/blob/d4b653e3bfbef7b3b4dcc4fe3bcae1e6fe07c8f6/scripts/validate_db_data_with_read_schemas.py#L74
and fails at this one https://github.com/fractal-analytics-platform/fractal-server/blob/d4b653e3bfbef7b3b4dcc4fe3bcae1e6fe07c8f6/scripts/validate_db_data_with_read_schemas.py#L80

The printed error is indeed that, but it is shown several lines up, between the Projects and Tasks
```sh
Project 49 validated
Project 50 validated
Project 53 validated
Project 56 validated
Traceback (most recent call last):
  File "/home/runner/work/fractal-server/fractal-server/scripts/validate_db_data_with_read_schemas.py", line 80, in <module>
    ApplyWorkflowRead(**job.model_dump())
  File "pydantic/main.py", line 341, in pydantic.main.BaseModel.__init__
pydantic.error_wrappers.ValidationError: 3 validation errors for ApplyWorkflowRead
workflow_dump -> timestamp_created
  field required (type=value_error.missing)
input_dataset_dump -> timestamp_created
  field required (type=value_error.missing)
output_dataset_dump -> timestamp_created
  field required (type=value_error.missing)
Task 1 validated
Task 2 validated
Task 3 validated
Task 4 validated
```

**Comment by ychiucco** [comment]
Maybe related to this issue

https://github.com/actions/runner/issues/664#issuecomment-1587153769

**Comment by tcompa** [comment]
Closed with #1225
